Stripe/STPPaymentMethodsInternalViewController.m

```diff
@@ -123,8 +123,9 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     cell.backgroundColor = self.theme.secondaryBackgroundColor;
     if (indexPath.section == STPPaymentMethodCardListSection) {
         id<STPPaymentMethod> paymentMethod = [self.paymentMethods stp_boundSafeObjectAtIndex:indexPath.row];
-        cell.imageView.image = paymentMethod.image;
+        cell.imageView.image = paymentMethod.templateImage;
         BOOL selected = [paymentMethod isEqual:self.selectedPaymentMethod];
+        cell.imageView.tintColor = selected ? self.theme.accentColor : [self.theme.primaryForegroundColor colorWithAlphaComponent:0.6f];
```

Comment:
Since this color is now used in 2 places in this VC (it's also used in `buildAttributedStringForCard:selected`) can you pull it into a helper method in this class?